src/iam/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(TARGET aosiamcpp)
+
+# ######################################################################################################################
+# Sources
+# ######################################################################################################################
+
+set(SOURCES certhandler/certhandler.cpp)
```

Comment:
i think it could be better to use 
`file(GLOB_RECURSE SOURCES CONFIGURE_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp")`

in this case it will be easier to add new source files